serverless.yml

```diff
@@ -0,0 +1,46 @@
+service: bitbucket-2fa-monitor
+
+provider:
+  name: aws
+  runtime: go1.x
+  stage: prod
+  timeout: 600
+  versionFunctions: false
+  memorySize: 512
+  region: us-east-1
+
```

Comment:
I think you'll need this in there as well (same indentation as region)



  iamRoleStatements:
    - Effect: "Allow"
      Action:
        - "ses:SendEmail"
      Resource: "*"

Reply:
Added.